test/functional/data/invalid_txs.py

```diff
@@ -21,6 +21,7 @@
 """
 import abc
 
+from typing import Optional
```

Comment:
This means that mypy is a requirement to run the tests, right?

Reply:
Ah no, this is the standard library: https://docs.python.org/3/library/typing.html

Reply:
Yes. Should I mention it in some documentation?

Reply:
Right